### ISSUE TITLE
undefined color

### ISSUE THREAD

**Issue by ArnoldSmith86** [ArnoldSmith86/virtualtabletop#1476]
It's possible that colors are undefined and then this happens:

```
ERROR - GENERIC player Guest123 connected to room example - TypeError: Cannot read properties of undefined (reading 'slice')
    at Room.newPlayerColor (file:///root/virtualtabletop/server/room.mjs:609:30)
    at Room.addPlayer (file:///root/virtualtabletop/server/room.mjs:34:52)
    at file:///root/virtualtabletop/server.mjs:321:31
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by 96LawDawg** [comment]
Would #1443 fix this?  The way 1443 works, I think, would now always force the color to be black in that situation.  Unless that error happens in such a way that the color routine never even gets called?

**Comment by 96LawDawg** [comment]
Closed by #1443

**Comment by robartsd** [comment]
I'm not sure if #1443 fixed this. I think toHex() will fail on the server at document.createElement('canvas') as the server has no document object.
